src/components/Drawer.js

```diff
@@ -152,11 +159,11 @@ const Drawer = React.createClass({
         backgroundColor: StyleConstants.Colors.PORCELAIN,
         boxShadow: StyleConstants.ShadowHigh,
 
-        '@media (max-width: 750px)': {
+        [`@media (max-width: ${this.props.breakPoints.medium}px)`]: {
```

Comment:
I'm not sure if Ryan is talking about the square brackets or the curly brackets. Why are the square brackets needed?